rest-api/src/main/java/gov/cms/qpp/conversion/api/services/CpcFileServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package gov.cms.qpp.conversion.api.services;
+
+import gov.cms.qpp.conversion.api.model.Metadata;
+import gov.cms.qpp.conversion.api.model.UnprocessedCpcFileData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Service for handling Cpc File meta data
+ */
+@Service
+public class CpcFileServiceImpl implements CpcFileService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)

Reply:
Go home Sonar, you're drunk.

Reply:
I don't think I need to change this. I don't think we want a constructor for a Service implementation.